dashboard/client/src/components/ActorTable.tsx

```diff
@@ -297,6 +313,36 @@ const ActorTable = ({
             ),
           }}
         />
+        <TextField
+          style={{ margin: 8, width: 120 }}
+          label="Class"
+          size="small"
+          InputProps={{
+            onChange: ({ target: { value } }) => {
+              changeFilter("actorClass", value.trim());
+            },
+            endAdornment: (
+              <InputAdornment position="end">
+                <SearchOutlined />
+              </InputAdornment>
+            ),
+          }}
+        />
+        <TextField
+          style={{ margin: 8, width: 120 }}
+          label="Repr"
```

Comment:
nit: I wonder if `repr` should be lowercased? I don't think `Repr` is an actual word.